<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="SideBar" name="widget" native="true"/>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedPane">
      <property name="currentIndex">
       <number>4</number>
      </property>
      <widget class="QWidget" name="_0_splash">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="logo">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resource.qrc">:/icons/icon0</pixmap>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="_1_login">
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>Login</string>
          </property>
          <property name="textFormat">
           <enum>Qt::TextFormat::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="_2_assets">
       <layout class="QFormLayout" name="formLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>Assets</string>
          </property>
          <property name="textFormat">
           <enum>Qt::TextFormat::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="_3_tickets">
       <layout class="QFormLayout" name="formLayout_3">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>Tickets</string>
          </property>
          <property name="textFormat">
           <enum>Qt::TextFormat::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="_4_steps">
       <layout class="QFormLayout" name="formLayout_4">
        <item row="0" column="0" colspan="2">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="locale">
           <locale language="English" country="UnitedStates"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::Box</enum>
          </property>
          <property name="text">
           <string>Steps</string>
          </property>
          <property name="textFormat">
           <enum>Qt::TextFormat::PlainText</enum>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMain">
    <property name="title">
     <string>Main</string>
    </property>
    <addaction name="actionLogin"/>
    <addaction name="actionAsset"/>
    <addaction name="actionTicket"/>
    <addaction name="actionStep"/>
   </widget>
   <addaction name="menuMain"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLogin">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </action>
  <action name="actionAsset">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Asset</string>
   </property>
  </action>
  <action name="actionTicket">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Ticket</string>
   </property>
  </action>
  <action name="actionStep">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Step</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SideBar</class>
   <extends>QWidget</extends>
   <header>sidebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
